src/app/lib/utilities/ichefURL/index.js

```diff
@@ -38,13 +39,8 @@ const buildIChefURL = ({ originCode, locator, resolution, isWebP = false }) => {
     .filter(Boolean)
     .join('/');
 
-  const isWebPSupported = isWebP && WEBP_ORIGIN_CODES.includes(originCode);
-
-  if (isWebPSupported) {
-    return `${url}.webp`;
-  }
-
-  return url;
+  return url.endsWith('.webp') ? url : `${url}.webp`;
+  return `${url}.webp`;
```

Comment:
The second return won't run, so needs to be deleted
```suggestion
  return url.endsWith('.webp') ? url : `${url}.webp`;
```

Reply:
return removed